pkg/receiptsmigration/migration.go

```diff
@@ -44,14 +44,14 @@ func Done(newPaths environment.Paths) (bool, error) {
 	if err != nil {
 		return false, err
 	}
-	store, err := ioutil.ReadDir(newPaths.InstallPath())
+	plugins, err := ioutil.ReadDir(newPaths.BinPath())
 	if err != nil {
 		return false, err
 	}
 
 	var hasInstalledPlugins bool
-	for _, entry := range store {
-		if entry.IsDir() {
+	for _, entry := range plugins {
+		if strings.HasPrefix(entry.Name(), "kubectl-") && 0 != entry.Mode()&os.ModeSymlink {
```

Comment:
what if we didn't look for kubectl- prefix?
similarly, what if we didn't look for them to be symlinks?
we can just ReadDir?

Reply:
That should be fine, unless somebody puts something into the `bin` directory. Doing these checks  makes that much more unlikely. But this should matter in very exceptional cases only.

Reply:
Yeah I guess it's especially fine because we'll remove this code ≥0.5 or maybe earlier.